### PR TITLE
 Changes to use log2timeline style guide and removed unused methods #176

### PR DESCRIPTION
**Pull request by joachimmetz**
Changes to use log2timeline style guide and removed unused methods #176

### CODE REVIEW

File: dftimewolf/lib/processors/turbinia.py [hunk 1]
```diff
@@ -107,54 +109,6 @@ def _print_task_data(self, task):
         continue
       print('   ' + path)
 
-  def display_task_progress(
```
Comment (joachimmetz) [inline dftimewolf/lib/processors/turbinia.py]: Note that this method was not used.

File: dftimewolf/lib/timesketch_utils.py [hunk 2]
```diff
@@ -98,69 +101,48 @@ def upload_timeline(self, timeline_name, plaso_storage_path):
...
+      sketch_id (int): identifier of the sketch on the Timesketch server.
+      index_id (int): identifier of timeline on the Timesketch server to
+          add to sketch.
     """
     resource_url = '{0:s}/sketches/{1:d}/timelines/'.format(
         self.api_base_url, sketch_id)
     form_data = {'timeline': [index_id]}
     self.session.post(resource_url, json=form_data)
 
-  def get_sketch(self, sketch_id):
```
Comment (joachimmetz) [inline dftimewolf/lib/timesketch_utils.py]: Note that this method was not used.

File: dftimewolf/lib/module.py [hunk 3]
```diff
@@ -5,40 +5,40 @@
 
 
 class BaseModule(object):
-  """Base class for modules.
+  """Interfact of a DFTimewolf module.
```
Comment (tomchop) [inline dftimewolf/lib/module.py]:
```suggestion
  """Interface of a DFTimewolf module.
```

**Comment by tomchop** [comment]
There also seem to be a few conflicts left, please resolve otherwise I'll take a look at them later.

File: dftimewolf/lib/module.py (hunk 3 above)
Comment (joachimmetz) [reply dftimewolf/lib/module.py]: ack

**Comment by joachimmetz** [comment]
@tomchop PTAL

**Pull request merged**